libbeat/scripts/Makefile

```diff
@@ -293,7 +293,7 @@ endif
 	@# Generate Kibana index pattern
 	@mkdir -p $(PWD)/_meta/kibana/5.x/index-pattern
 	@mkdir -p $(PWD)/_meta/kibana/default/index-pattern
-	@${PYTHON_ENV}/bin/python ${ES_BEATS}/libbeat/scripts/generate_index_pattern.py --index '${BEAT_INDEX_PREFIX}-*' --beat-name ${BEAT_NAME} --beat $(PWD) --version ${BEAT_VERSION}
+	go run -v ${ES_BEATS}/libbeat/scripts/generate_index_pattern.go '${BEAT_INDEX_PREFIX}-*' ${BEAT_NAME} $(PWD) ${BEAT_VERSION}
```

Comment:
not sure I understand you correctly, I just moved the generation from a `python` to `go` implementation, but did not change the directory structure..

Reply:
Good point. The difference is here in how we handle go "scripts" and all other scripts. Because in general only 1 `main` function should be in a package we have under `dev-tools/cmd` for each go script a package. The above will work as intended. We can still move it later.